src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services9_compiled.php

```diff
@@ -246,7 +246,7 @@ protected function getMethodCall1Service()
 
         $instance->setBar($this->get('foo'));
         $instance->setBar(NULL);
-        $instance->setBar(($this->get("foo")->foo() . (($this->hasparameter("foo")) ? ($this->getParameter("foo")) : ("default"))));
+        $instance->setBar(($this->get('foo')->foo().(($this->hasParameter('foo')) ? ($this->getParameter('foo')) : ('default'))));
```

Comment:
As said in my other comment, this is a bug in fabbot. It should not check fixtures files. Please just ignore the failures coming from fixtures files